Sources/Commands/Options.swift

```diff
@@ -328,6 +328,15 @@ public struct SwiftToolOptions: ParsableArguments {
         help: "Specify the .netrc file path.",
         completion: .file())
     var netrcFilePath: AbsolutePath?
+    
+#if canImport(Security)
+    /// Whether to use keychain for authenticating with remote servers
+    /// when downloading binary artifacts or communicating with a registry.
+    @Flag(inversion: .prefixedNo,
```

Comment:
For `.netrc` handling, we settled on `--enable` / `--disable` style inversion, so I think we should be consistent across these two options (whichever convention we decide).

Reply:
I think enable/disable is preferable.